src/common/repo.ts

```diff
@@ -0,0 +1,580 @@
+    },
+    {
+      kind: 'AplDns',
+      envDir,
+      jsonPathExpression: '$.dns',
+      pathGlob: `${envDir}/env/settings/*dns.{yaml,yaml.dec}`,
+      processAs: 'mapItem',
+      resourceGroup: 'platformSettings',
+      resourceDir: 'settings',
+      loadToSpec: true,
+    },
+    {
+      kind: 'AplIngress',
+      envDir,
+      jsonPathExpression: '$.ingress',
+      pathGlob: `${envDir}/env/settings/ingress.yaml`,
+      processAs: 'mapItem',
+      resourceGroup: 'platformSettings',
+      resourceDir: 'settings',
+      loadToSpec: true,
+    },
+    {
+      kind: 'AplKms',
+      envDir,
+      jsonPathExpression: '$.kms',
+      pathGlob: `${envDir}/env/settings/*kms.{yaml,yaml.dec}`,
+      processAs: 'mapItem',
+      resourceGroup: 'platformSettings',
+      resourceDir: 'settings',
+      loadToSpec: true,
+    },
+    {
+      kind: 'AplObjectStorage',
+      envDir,
+      jsonPathExpression: '$.obj',
+      pathGlob: `${envDir}/env/settings/*obj.{yaml,yaml.dec}`,
+      processAs: 'mapItem',
+      resourceGroup: 'platformSettings',
+      resourceDir: 'settings',
+      loadToSpec: true,
+    },
+    {
+      kind: 'AplIdentityProvider',
+      envDir,
+      jsonPathExpression: '$.oidc',
+      pathGlob: `${envDir}/env/settings/*oidc.{yaml,yaml.dec}`,
+      processAs: 'mapItem',
+      resourceGroup: 'platformSettings',
+      resourceDir: 'settings',
+      loadToSpec: true,
+    },
+    {
+      kind: 'AplCapabilitySet',
+      envDir,
+      jsonPathExpression: '$.otomi',
+      pathGlob: `${envDir}/env/settings/*otomi.{yaml,yaml.dec}`,
+      processAs: 'mapItem',
+      resourceGroup: 'platformSettings',
+      resourceDir: 'settings',
+      loadToSpec: true,
+    },
+    {
+      kind: 'AplBackupCollection',
+      envDir,
+      jsonPathExpression: '$.platformBackups',
+      pathGlob: `${envDir}/env/settings/*platformBackups.{yaml,yaml.dec}`,
+      processAs: 'mapItem',
+      resourceGroup: 'platformBackups',
+      resourceDir: 'settings',
+      loadToSpec: true,
+    },
+    {
+      kind: 'AplSmtp',
+      envDir,
+      jsonPathExpression: '$.smtp',
+      pathGlob: `${envDir}/env/settings/*smtp.{yaml,yaml.dec}`,
+      processAs: 'mapItem',
+      resourceGroup: 'platformSettings',
+      resourceDir: 'settings',
+      loadToSpec: true,
+    },
+    {
+      kind: 'AplUser',
+      envDir,
+      jsonPathExpression: '$.users[*]',
+      pathGlob: `${envDir}/env/users/*.{yaml,yaml.dec}`,
+      processAs: 'arrayItem',
+      resourceGroup: 'users',
+      resourceDir: 'users',
+      loadToSpec: true,
+    },
+    {
+      kind: 'AplVersion',
+      envDir,
+      jsonPathExpression: '$.versions',
+      pathGlob: `${envDir}/env/settings/versions.yaml`,
+      processAs: 'mapItem',
+      resourceGroup: 'platformSettings',
+      resourceDir: 'settings',
+      loadToSpec: true,
+    },
+    {
+      kind: 'AplTeamCodeRepo',
+      envDir,
+      jsonPathExpression: '$.teamConfig.*.coderepos[*]',
```

Comment:
I think I wrote this but I changed it in api to:
```suggestion
      jsonPathExpression: '$.teamConfig.*.codeRepos[*]',
```